oshi-core/src/main/java/oshi/software/os/OSProcess.java

```diff
@@ -523,4 +524,24 @@ public void setBytesRead(long bytesRead) {
     public void setBytesWritten(long bytesWritten) {
         this.bytesWritten = bytesWritten;
     }
+    
+    /**
+     * Sets the number of open file handles (or network connections) that
+     * belongs to the process
+     * since .5.0
+     * @param count 
+     */
+    public void setOpenFiles(long count){
+        this.handles = count;
+    }
+    
+    /**
+     * Sets the number of open file handles (or network connections) that 
+     * belongs to the process
+     * @since 3.5.0

```

Comment:
While this may be good documentation we've not used it at all in the past; also this may be a 3.4.x patch release....

Reply:
ok, what should this return if it's not supported or could not be acquired? -1? 0?

Reply:
I would suggest either deleting the @since, or updating all the rest of the code to add the @since everywhere it's needed (which is pretty much everywhere).